setup.py

```diff
@@ -702,14 +715,19 @@ def readme():
 
 
 install_requires = [
-    'attrdict >= 2.0.0',
+    'attridict >= 0.0.8',
+    #'attridict >= 0.0.8; python_version >= "3.10"', #replacement for attrdict
+    #'attrdict >= 2.0.0; python_version < "3.10"', #replacement for attrdict
```

Comment:
It looks like you can remove the commented out code

Reply:
Yea I've removed attrdict. will remove